### PR TITLE
Update CoffeeScript to 1.12.6

### PR DESCRIPTION
**Pull request by GeoffreyBooth**
Bumps the CoffeeScript version, and uses the new primary repo, `coffeescript`.

### CODE REVIEW

**Comment by GeoffreyBooth** [comment]
@hwillson Thanks. When will the new package be released?

File: packages/non-core/coffeescript/package.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
 Package.describe({
   summary: "Javascript dialect with fewer braces and semicolons",
-  version: "1.12.3_1"
+  version: "1.12.6_1"
```
Comment (laosb) [inline packages/non-core/coffeescript/package.js]: Shall we remove the `_1` in the version number, since we're releasing for 1.12.6 for the first time?

**Comment by hwillson** [comment]
I've added a `History.md` entry and adjusted the package version. As soon as the tests finish running, I'll merge. Thanks!

**Comment by hwillson** [comment]
Well, Travis appears to be hung. The build passed but Travis hasn't relayed those details to GH. I'm looking into it ...

File: packages/non-core/coffeescript/package.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Package.describe({
   summary: "Javascript dialect with fewer braces and semicolons",
-  version: "1.12.6_1"
+  version: "1.12.6"
```
Comment (mitar) [inline packages/non-core/coffeescript/package.js]: No! Don't do that! Please wrap all upstream packages always with `_1` and then bump it if needed. It is really confusing if there is `1.12.6` and later on `1.12.6_1`. Which one is newer? Hard to know from just looking at the version.

**Comment by abernix** [comment]
Published as `coffeescript@1.12.6_1`!  Thanks!

**Pull request merged**